src/modules/navigator/mission_block.cpp

```diff
@@ -251,15 +251,26 @@ MissionBlock::is_mission_item_reached()
 				}
 
 			} else {
+				// Determine altitude acceptance radius. By default, this is taken from the respective parameter.
+				// However, before a landing approach the acceptance radius needs to be tighter: Assume a 30% error
+				// w.r.t. the remaining descent altitude is OK, but enforce min/max values (e.g. min=3m to make
+				// sure that the waypoint can still be reached in case of wrong user input).
+				float alt_accept_rad = _navigator->get_altitude_acceptance_radius();
+				struct position_setpoint_s next_sp = _navigator->get_position_setpoint_triplet()->next;
+
+				if (next_sp.type == position_setpoint_s::SETPOINT_TYPE_LAND && next_sp.valid) {
+					alt_accept_rad = math::constrain(0.3f * (curr_sp->alt - next_sp.alt), 3.0f,
+									 _navigator->get_altitude_acceptance_radius());
+					PX4_INFO("Accept:%5.3f", double(alt_accept_rad));
```

Comment:
Fine for debugging, but can you drop these messages before merge? PX4_INFO isn't even logged.

Reply:
Changed, must have slipped through somehow...